public/index.html

```diff
@@ -18,107 +22,292 @@
     </div>
   </header>
 
   <nav role="navigation" class="nav">
     <div class="nav__box">
-      <a class="nav__link" href="">The problem</a>
-      <a class="nav__link" href="">Key factors</a>
-      <a class="nav__link" href="">The question</a>
-      <a class="nav__link" href="">The solution</a>
+      <a class="nav__link--problem red-nav" href="#problem-section">The problem</a>
+      <a class="nav__link--keyFactors red-nav" href="#keyFactors-section">Key factors</a>
+      <a class="nav__link--question red-nav" href="#question-section">The question</a>
+      <a class="nav__link--solution blue-nav" href="#solution-section">The solution</a>
     </div>
   </nav>
 
-  <section role="region" class="problem">
+  <section id="problem-section" class="problem">
     <div class="problem__wrapper">
-      <p class="problem__intro">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.
+      <h2 class="screen-reader">The problem</h2>
+      <p class="problem__intro intro-text">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.
       </p>
-      <p class="problem__context">Just in the U.S., non-adherence has been attributed to over $100 billion dollars in avoidable healthcare costs.<sup class="problem__sup">1</sup> In developed countries, among patients with chronic illness, approximately 50% do not take medications
-        as prescribed.<sup class="problem__sup">2</sup>
+      <p class="problem__context context-text">Just in the U.S., non-adherence has been attributed to over $100 billion dollars in avoidable healthcare costs.
+        <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3934668/"><sup class="problem__sup">1</sup></a> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.
+        <a href="http://www.who.int/"><sup class="problem__sup">2</sup></a>
       </p>
-      <blockquote class="problem__quote" cite="">
+      <blockquote class="problem__quote" cite="http://apps.who.int/medicinedocs/en/d/Js4883e/5.html">
         <p>"Increasing the effectiveness of adherence interventions might have a greater impact on the health of the population than any improvement in specific medical treatment."</p>
       </blockquote>
-      <cite class="problem__cite">World Health Organization, 2005<sup class="problem__sup">3</sup></cite>
-
-      <img class="problem__bottle1 icon" src="src/icons/pill-bottle3.svg">
-      <img class="iproblem__bottle2 icon" src="src/icons/section2-illo.svg">
-      <img class="problem__bottle3 icon" src="src/icons/pill-bottle3.svg">
-      <img class="problem__pills1 icon" src="src/icons/purple-pills.svg">
+      <cite class="problem__author">World Health Organization, 2005
+        <a href="http://www.who.int/"><sup class="problem__sup">3</sup></a>
+      </cite>
```

Comment:
Same indentation note here - I missed that `World Health Organization, 2005` text for a minute. This is a little better:

```
<cite class="problem__author">
  World Health Organization, 2005
  <a href="http://www.who.int/"><sup class="problem__sup">3</sup></a>
</cite>
```